tests/resources/functional/snowflake/core_engine/lca/lca_homonym.sql

```diff
@@ -0,0 +1,27 @@
+-- snowflake sql:
+select
+  ca_zip
+from (
+  SELECT
+    substr(ca_zip,1,5) ca_zip,
```

Comment:
This is a nice edge-case where the column alias has the same name as an existing column.